site/content/contribute/team_contributions/_index.md

```diff
@@ -25,9 +25,13 @@ Join others who have contributed code to Mattermost repositories in GitHub, and
+
+Join others who have contributed an integration, and [get started today](https://developers.mattermost.com/integrate/getting-started/).
+
+Accxia GmbH (2019.08.17), Adnan Hajdarević (2015.03.13), Affirm (2018.11.18), Agoston Szepessy (2017.07.15), Alex Tzonkov (2019.03.19), Alexander Meindl, AlphaNodes (2016.02.11), Alexey Korolev (2018.08.02), AltSol (2016.02.11), Amit Yadav (2019.04.27), Andy Lo-A-Foe (2016.02.13), Anton Kiryushkin (2018.12.03), Ben Schumacher (2018.08.20), bitbackofen (2015.11.15), Blindside Networks (2018.07.05), Brian Hopkins (2018.06.18), Brightscout, Inc (2019.03.04), bytemine GmbH (2019.04.18), Carlos Tadeu Panato Junior (2019.02.25), Carsten Seeger (2019.06.30), Cédric Brun (2016.01.05), Charlie Pinto, CloudApp (2018.01.05), Chocolatey Community (2016.08.29), Chris Herborth (2018.11.12), Christian Johannsen (2019.04.29), Christian Plümer (2017.06.26), Christopher De Cairos (2016.09.16), Christopher Hranj (2018.07.17), Clémence Mousset (2018.02.02), Claudio Costa (2019.08.05), Cloud Foundry Community (2017.03.15), Cloudron (2016.04.19), Craig Vitter (2018.06.07), Daniel Schalla (2019.01.13), David Duarte (2017.08.28), David John Coleman II, PagerDuty (2018.09.17), David Taylor (2017.06.25), Eliot Berriot (2018.01.17), Emanuele Panzeri (2017.07.10), Evan do Carmo (2019.05.09), Felix Bartels (2017.11.20), Frederic Guillot (2015.11.02), Galois, Inc (2017.02.02), gotlium (2016.02.14), Harshil Sharma (2019.04.27), Healthchecks (2019.08.22), Herzum (2019.04.15), Homebrew (2016.12.15), Hybrid Chat (2019.03.18), InfraCloud Technologies (2019.04.04), Jed Fonner (2017.12.17), JenkinsCI (2015.10.23), Jesús Espino (2019.02.02), Kolin Keane (2018.05.16), Kopano (2017.12.22), kosgrz (2019.05.30), Kyâne Pichou (2016.12.14), Luca Agnello (2017.04.22), Lujeni, Numberly (2016.03.24), Łukasz Dziedziul (2017.12.25), Mario Steinitz, Shoreless (2017.06.21), Markus Busche (2017.08.19), Markus Mueller (2016.03.10), Martin Kraft (2019.01.13), Matthew Dorner (2020.03.02), Matthieu Brouillard (2018.01.15), Michael K. Sondej (2019.08.18), Miguel de la Croix (2019.01.13), Nafis Faysal (2018.12.22), Ndrive (2015.12.11), NotSqrt (2016.12.19), Orbital Impact (2018.10.18), Pan Luo (2016.12.14), Patrick Liu (2018.04.20), Paul Rothrock (2018.02.16), Ricardo Tribaldos (2018.05.24), Richard Grainger (2016.01.09), Rodrigo Corsi (2019.02.09), Romain Maneschi (2019.02.27), Sameroom (2016.11.03), Scott Lee Davis (2018.07.05), Sean Sackowitz (2018.03.29), Serhat Can, Opsgenie (2017.01.02), SoftDevTeam (2016.01.19), Steven Fluck, Ergon (2019.03.04), Takayuki Maruyama (2018.01.08), thepill (2019.04.30), Thibaud Dauce (2016.08.29), Tyler Tomlinson (2015.11.02), Wasim Thabraze (2019.03.17), William Ernest (2019.03.10), Wim (2015.10.23), Yi EungJun (2016.12.14), Yusuke Nemoto (2018.02.02), Zabbix (2018.11.08), Zarko Jevtic (2018.02.11)
```

Comment:
Just a note on this: five entries don't include the contribution date. They are: Alexander Meindl, Charlie Pinto, David John Coleman II, Mario Steinitz, Serhat Can.

Reply:
@justinegeffen They were actually made as part of the company listed after them, but I agree this is not clear. I've changed the formatting to use `Alexander Meindl - AlphaNodes (2016.02.11)` instead of using a comma between the contributor's name and the company